### ISSUE TITLE
Scss-lint column numbers

### ISSUE THREAD

**Issue by bignimbus** [scrooloose/syntastic#1660]
I want to be able to highlight warnings inline with Scss-lint like I do with Eslint and Rubocop.  See the difference below:

### Eslint: error shows in gutter and on line/column
<img width="718" alt="screen shot 2016-01-08 at 1 09 58 pm" src="https://cloud.githubusercontent.com/assets/3976692/12206903/3fd1f4c6-b609-11e5-9c55-c0fd5fa24a3e.png">

#### but I don't like the gutter :(
<img width="719" alt="screen shot 2016-01-08 at 1 10 32 pm" src="https://cloud.githubusercontent.com/assets/3976692/12206907/41c90044-b609-11e5-87d3-2d1137f91877.png">

Here's the debug output:
```
"test.scss" 4L, 36C
syntastic: 0.094294: &shell = '/bin/bash', &shellcmdflag = '-c', &shellpipe = '2>&1| tee
', &shellquote = '', &shellredir = '>%s 2>&1', &shellslash = 0, &shelltemp = 1, &shellxq
uote = '', &shellxescape = ''
syntastic: 0.094689: UpdateErrors (auto): default checkers
syntastic: 0.094967: CacheErrors: default checkers
syntastic: 0.095541: g:syntastic_aggregate_errors = 0
syntastic: 0.095701: getcwd() = '/Users/jeffreyauriemma/personal/game-of-life-n'
syntastic: 0.617293: CacheErrors: Invoking checker: scss/scss_lint
syntastic: 0.617819: SyntasticMake: called with options: {'errorformat': '%f:%l [%t] %m'
, 'makeprg': 'scss-lint test.scss', 'returns': [0, 1, 2, 65, 66]}
syntastic: 0.975939: checker output: ['test.scss:2 [W] PropertySpelling: Unknown propert
y silly-error', '']
syntastic: 0.976459: raw loclist: [{'lnum': 2, 'bufnr': 1, 'col': 0, 'valid': 1, 'vcol':
 0, 'nr': -1, 'type': 'W', 'pattern': '', 'text': 'PropertySpelling: Unknown property si
lly-error'}, {'lnum': 0, 'bufnr': 0, 'col': 0, 'valid': 0, 'vcol': 0, 'nr': -1, 'type':
'', 'pattern': '', 'text': ''}]
syntastic: 0.976817: getLocList: checker scss/scss_lint returned 1
syntastic: 0.976966: scss/scss_lint raw: [{'lnum': 2, 'bufnr': 1, 'col': 0, 'valid': 1,
'vcol': 0, 'nr': -1, 'type': 'W', 'pattern': '', 'subtype': 'Style', 'text': 'PropertySp
elling: Unknown property silly-error'}, {'lnum': 0, 'bufnr': 0, 'col': 0, 'valid': 0, 'v
col': 0, 'nr': -1, 'type': '', 'pattern': '', 'subtype': 'Style', 'text': ''}]
syntastic: 0.977243: quiet_messages filter: {}
syntastic: 0.977782: aggregated: {'_sorted': 0, '_name': 'scss_lint (scss)', '_owner': 1
, '_columns': 1, '_rawLoclist': [{'lnum': 2, 'bufnr': 1, 'col': 0, 'valid': 1, 'vcol': 0
, 'nr': -1, 'type': 'W', 'pattern': '', 'subtype': 'Style', 'text': 'PropertySpelling: U
nknown property silly-error'}]}
```

Since column data is being collected, I wonder if this could be solved by simply formatting the output differently.

Any thoughts on this?

**Comment by lcd047** [comment]
> Since column data is being collected

Not sure how did you reach this conclusion; it currently isn't.

It _could_ be, since `scss-lint` has added a JSON reporter in version 0.29.0, and that reporter _does_ output column numbers.  But changing syntastic to take advantage of that involves writing a special parser.  That isn't a huge problem, but I don't really have time for that right now.  Perhaps in a few days.

**Comment by bignimbus** [comment]
@lcd047 https://github.com/brigade/scss-lint/pull/671 looks like there's work happening on their side to update the default reporter.

**Comment by lcd047** [comment]
It doesn't matter, the JSON output is available since version 0.29.0, and it offers more features.  Please see commit 2db6f53.
